build.gradle

```diff
@@ -52,6 +52,7 @@ buildscript {
       javaStringSimilarity: '1.1.0',
       espresso            : '3.1.1',
       leakCanary          : '1.6.2',
+      constraintLayout    : '1.1.2',
```

Comment:
The latest version of `androidx.constraintlayout` is `1.1.3`. Let's bump this up?

Reply:
We'll do this later. Let's favor merging branches asap and adding updates to backlog.

Reply:
Considering that this is PR for an ongoing, changes like these can go in upcoming PRs